azure-pipelines.yml

```diff
@@ -0,0 +1,63 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  batch: true
```

Comment:
No need of batch tasks this time, delete please

Reply:
ok 
it's done